internal/datanode/data_node.go

```diff
@@ -1169,6 +1169,12 @@ func importFlushReqFunc(node *DataNode, req *datapb.ImportTaskRequest, res *root
 			rowNum = field.RowNum()
 			break
 		}
+
+		if rowNum <= 0 {
+			log.Info("fields data is empty, no need to generate segment")
```

Comment:
Need to be more verbose, consider including task ID, shard ID, # of channels, channel name, etc.